src/pybind/mgr/cephadm/services/nvmeof.py

```diff
@@ -67,11 +84,28 @@ def get_set_cmd_dicts(out: str) -> List[dict]:
 
             for dd in daemon_descrs:
                 assert dd.hostname is not None
+                service_name = dd.service_name()
 
                 if not spec:
-                    logger.warning(f'No ServiceSpec found for {dd.service_name()}')
+                    logger.warning(f'No ServiceSpec found for {service_name}')
                     continue
 
+                self.mgr.check_mon_command({
+                    'prefix': 'config-key set',
+                    'key': f'mgr/dashboard/{service_name}/mtls_server_cert',
```

Comment:
I was wondering if instead of storing certificates under the `mgr/dashboard/...` prefix, shouldn't we store them centralized (not only NVMe's, but all of then) on a generic location for certificates (`secrets/certificates`). That would also simplify managing the certificate lifecycle (renewal, revokation, ...). CC: @rkachach

Reply:
@nizamial09 @epuertat indeed, Adam introduced recently some changes to centralize the certs/keys mgmt on cephadm (https://github.com/ceph/ceph/pull/55564). Moving forward all the certificates that are needed by any service must be stored and managed by cephadm. As Ernesto stated among other benefits that would make it easier to keep track of all the certificates and in the short term use some secure-store for the certificates instead of the mon-store.

Reply:
@rkachach i am now getting the certificates from the store with the latest updates.